packages/vue/src/mdx-hast-to-vue-jsx.js

```diff
@@ -1,105 +0,0 @@
-function toVueJSX(node, parentNode = {}, options = {}) {
```

Comment:
I deleted these files because they weren't being utilized. 🙏🏽 Please let me know if you think I should keep them. I figured we could recover them with git.